courses.html

```diff
@@ -274,7 +274,7 @@ <h1 class="text-white">Our Courses</h1>
         </div>
         <div class="row" style="width: 100%">
           <div class="col-lg-12 col-md-12 col-sm-12 text-center mb-3" style="font-size: 0.9em">
-            &copy;2021 ACM DBIT Student Chapter.<br />Developed by Grejo Joby
+            &copy;2022 ACM DBIT Student Chapter.<br />Developed by Grejo Joby
```

Comment:
https://github.com/flockflair-dbit/acm-website/pull/11/files#diff-9e0b29b3271249d4b2874eba09617a21796c0a35650a3bf666a312813470aaacR517

use the footer from index page